### PR TITLE
chore: add knobs to button component

### PR DESCRIPTION
**Pull request by HerrBertling**

### CODE REVIEW

File: packages/evergarden/src/Button/examples.js [hunk 1]
```diff
@@ -1,97 +1,48 @@
...
+  },
+  template: `
+  <Button
+    :variantColor="variant"
+    :size="size"
+    :isDisabled="disabled"
+    onClick={action('hello')
+  >
+    {{ text }}
+  </Button>`
```
Comment (egoist) [inline packages/evergarden/src/Button/examples.js]: let's use `render` function instead 😄

Comment (HerrBertling) [reply packages/evergarden/src/Button/examples.js]: I really have no idea how to get that running with knobs 🙈

Comment (egoist) [reply packages/evergarden/src/Button/examples.js]: the template code you wrote is actually broken

Comment (HerrBertling) [reply packages/evergarden/src/Button/examples.js]: Oh, yes - the missing curly bracket. That‘s what happens when the baby suddenly wakes up and you push stuff in a hurry 😅

**Pull request merged**